shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/service/DatasetServiceImpl.java

```diff
@@ -507,6 +508,27 @@ public DatasetAcquisition getAcquisition(Dataset dataset) {
     }
 
     @Override
+    public OverallStatisticsDTO getOverallStatistics() {
```

Comment:
There could be good reason to use a DTO inside the service layer and I might not understand everything here but if it's not justified we must use the original class.

Reply:
Actually it seems it's just a naming issue sorry